cli/cli_app.go

```diff
@@ -235,6 +237,7 @@ GLOBAL OPTIONS:
    terragrunt-override-attr                     A key=value attribute to override in a provider block as part of the aws-provider-patch command. May be specified multiple times.
    terragrunt-debug                             Write terragrunt-debug.tfvars to working folder to help root-cause issues.
    terragrunt-log-level                         Sets the logging level for Terragrunt. Supported levels: panic, fatal, error, warn (default), info, debug, trace.
+   terragrunt-strict                            Sets strict mode for the validate-inputs command. By default, strict mode is off. When this flag is passed, strict mode is turned on. When strict mode is turned off, the validate-inputs command will only return an error if required inputs are missing from all input sources (env vars, var files, etc). When strict mode is turned on, an error will be returned if required inputs are missing OR if unused variables are passed to Terragrunt.
```

Comment:
I could use a second opinion here from @brikis98, but looking at this now, I wonder if this should be named something more specific to `validate-inputs`? E.g., `terragrunt-strict-validate`.

Otherwise, it's confusing as you might mix this up with the existing [--terragrunt-strict-include flag](https://terragrunt.gruntwork.io/docs/reference/cli-options/#terragrunt-strict-include).

Reply:
Makes sense - the same thought crossed my mind as well.

Reply:
Agreed! `--terragrunt-strict-validate` seems reasonable.

Reply:
Changed to `--terragrunt-validate-strict` in 9fc55d6